hera_mc/cm_active.py

```diff
@@ -205,23 +205,42 @@ def load_geo(self, at_date=None):
             key = cm_utils.make_part_key(ageo.station_name, None)
             self.geo[key] = ageo
 
-    def revs(self, hpn):
+    def revs(self, hpn, exact_match=False):
         """
-        Returns a list of active revisions for a given hpn.
+        Returns a list of active revisions for given hpn.
 
         Parameters
         ----------
-        hpn : str
-            A HERA part number (not a list)
+        hpn : str or list
+            A HERA part number, or list of part numbers or fractions thereof.
```

Comment:
this should specify it's the beginning that's matched (startswith) not any "fraction" of the part.

Reply:
Done.